librz/analysis/p/analysis_pyc.c

```diff
@@ -103,7 +103,11 @@ static int pyc_op(RzAnalysis *a, RzAnalysisOp *op, ut64 addr, const ut8 *data, i
 
 	if (op_obj->type & HASJABS) {
 		op->type = RZ_ANALYSIS_OP_TYPE_JMP;
-		op->jump = func_base + oparg;
+		op->jump = func_base;
+		if (ops->jump_use_instruction_offset)
+			op->jump += oparg * 2;
```

Comment:
is this always `*2` or does it depend on `RzBinInfo->bits / 8` value?

Reply:
it is always `*2`, it refers to the opcode size, which is always `2` since python 3.6